src/common/log.c

```diff
@@ -110,6 +118,31 @@ static ArdopLogFile* ALL_LOG_FILES[] = {
 	&SessionLog
 };
 
+#ifdef LOG_OUTPUT_SYSLOG
+/* Map zf_log level to syslog */
+static int log_level_syslog(const int lvl)
+{
+	switch (lvl)
+	{
+	case ZF_LOG_VERBOSE:
+		return LOG_DEBUG;
+	case ZF_LOG_DEBUG:
+		return LOG_DEBUG;
+	case ZF_LOG_INFO:
+		return LOG_INFO;
+	case ZF_LOG_WARN:
+		return LOG_WARNING;
+	case ZF_LOG_ERROR:
+		return LOG_ERR;
+	case ZF_LOG_FATAL:
+		return LOG_EMERG;
+	default:
+		assert(!"unreachable");
+		return LOG_EMERG;
```

Comment:
```suggestion
	case ZF_LOG_FATAL:
		return LOG_ALERT;
	default:
		assert(!"unreachable");
		return LOG_ALERT;
```

> So, I propose that we replace the few instances of ZF_LOGF() with ZF_LOGE()

That would certainly do the trick, but then we'd be sacrificing one of our log levels. An alternative would be to change the mapping here such that `LOG_EMERG` priority messages could not be emitted.

Reply:
I like your approach better, but did you intend 
```
case ZF_LOG_FATAL:
		return LOG_ERR;
```
where you wrote
```
case ZF_LOG_FATAL:
		return LOG_ALERT;
```

I think that it is reasonable to omit `LOG_PERROR` such that when using the `--syslog` option, behavior is appropriate for ardopcf running as a daemon.  For other uses, the existing logging system is sufficient.

I've complied and briefly tested the current contents of this pull request on Windows.  This confirms that your use of `#ifdef LOG_OUTPUT_SYSLOG` so that this change does not affect the Windows builds is working as intended.

You should update docs/Commandline_options.md to describe what this new option does and when it is appropriate to use it.  For improvements to documentation describing existing features, I've previously written that it is appropriate to submit pull requests to the master branch.  However, in this case of describing a change in behavior, it makes sense to include it as part of this pull request to the develop branch.

Reply:
I actually did mean `LOG_ALERT`, which has a higher priority than `LOG_ERR` but lower than `LOG_EMERG`. But, after reflection, I think `LOG_CRIT` is probably better. It sits just above `LOG_ERR`, so using it avoids a gap in the priority list. I think we do want a higher level than `LOG_ERR` for `ZF_LOG_FATAL` since it will help operators find and diagnose failure events.

43939741f58024b5f06ffeb23412a83129302d4d uses `LOG_CRIT`.

Priority values are described in [syslog(3)](https://www.man7.org/linux/man-pages/man3/syslog.3.html).